### DISCUSSION TITLE
ubuntu Python version how to change ?

### DISCUSSION THREAD

**Discussion by productdevbook** [obsproject/obs-studio#8412] [General]
![image](https://user-images.githubusercontent.com/38668796/223126251-b09549f4-2553-4bc5-a42e-5ddd756fa519.png)

Dont see path change

**Comment by tytan652** [comment]
Linux builds are bound to the system-wide python unlike macOS and Windows. So no way to change it, there is no configuration option in OBS on Linux.

**Reply by productdevbook** [reply]
but I uploaded a new version with brew, my ways are different. /home/linuxbrew/.linuxbrew/Cellar/python@3.11

**Reply by tytan652** [reply]
There is no configuration option in OBS, we link with the system Python.

Installing a new Python with homebrew will change nothing.
And replacing the system-wide one  provided by the distro packages with a newer one might break OBS.

**Comment by Fenrirthviti** [comment]
Installing versions of Python from brew are not officially supported by OBS.

We currently expect you to use the `python3-dev` package, which OBS should automatically detect and link to. For further assistance getting this set up since this is a support issue at this point, please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/).

Thank you!
